PV260_Project/Tests/BLTests/CsvFileDownloader.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using WireMock.ResponseBuilders;
+using WireMock.Server;
+using Request = WireMock.RequestBuilders.Request;
+using BusinessLayer.DataLoading;
+using DataLayer.Models;
+using FluentAssertions;
+
+namespace Tests.BLTests
+{
+    public class Tests
+    {
+        private DataSet _testFile;
+        private WireMockServer _server;
+        [SetUp]
+        public void Setup()
+        {
+            var body = File.ReadAllText("../../../TestFiles/csv_file_loader_test.csv");
```

Comment:
Maybe rename file to something more generic?

Reply:
Maybe the filename could mention that it contains holdings (not sure if that's going more generic or specific :D) ? But maybe that's already obvious. I would probably keep it as it is.
(just an obvious note: since the file already exists, I assume it's used in other tests, so the affected paths will have to be changed - unless, ofc, it's not used in any tests)

Reply:
It's used in CsvFileLoaderTests.cs (therefore the csv_file_loader_test.csv name). Isn't it weird, that file with such name is used in completely different test?
I personally have no problem with that but in evaluation from M. Kubus is critisized using UnitTest1.csv.
I will keep it like this if no one else objects.

Reply:
later when you will change filename don't forget to change class name as well .. now I see just "public class Tests"